src/test_workflow/README.md

```diff
@@ -110,6 +110,17 @@ To run OpenSearch Dashboards BWC tests.
 opensearch-dashboards=https://ci.opensearch.org/ci/dbc/distribution-build-opensearch-dashboards/1.3.5/4056/linux/x64/tar
 ```
 
+#### BWC tests on distribution bundle level
+
+The BWC tests running on distribution level are using the same framework from OpenSearch. The test cluster is spin up with the `latest` distribution bundle of provided version exclusively when the project is initialized with property `-PcustomDistributionDownloadType=bundle`.
+
+The distribution bundle will be obtained with this format URL: `https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/1.3.2/latest/linux/x64/tar/dist/opensearch/opensearch-1.3.2-linux-x64.tar.gz`
```

Comment:
```suggestion
The distribution bundle (for example tarball) will be obtained with this sample URL: `https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/1.3.2/latest/linux/x64/tar/dist/opensearch/opensearch-1.3.2-linux-x64.tar.gz`
```

Reply:
@gaiksaya Thanks for review. However, I think our BWC distribution test are hardcoded right now to only support tarball distribution because of the framework we used from the OpenSearch repo.

Reply:
I see! Maybe then just say:
```
Example distribution bundle URL: https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/1.3.2/latest/linux/x64/tar/dist/opensearch/opensearch-1.3.2-linux-x64.tar.gz
```